assests/CSS/mobileStyle.css

```diff
@@ -0,0 +1,558 @@
+  z-index: 2;
+}
+
+.mobile-Navlinks {
+  display: flex;
+  flex-direction: column;
+  gap: 7%;
+  height: 100vh;
+  width: 100%;
+  position: fixed;
+  left: 0;
+  top: 0;
+  padding: 86px 28px;
+  background: #474646;
+  font-family: 'lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 5vw;
+  line-height: 45%;
+  transition: all 0.5s;
+  z-index: 1;
+  list-style: none;
+}
+
+.mobile-Navlinks a {
+  color: #fff;
+}
+
+.mobile-Navlinks .logo {
+  font-weight: 700;
+  font-size: 5vw;
+  line-height: 60%;
+}
+
+.blur {
+  filter: blur(5px);
+}
+
+/* Headline section */
+.headline {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 5vw 5vw;
+  background-image: url(../Images/headline-background.png);
+}
+
+.headline .message {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  font-family: 'lato', sans-serif;
+  font-weight: 400;
+  font-size: 5vw;
+  line-height: 9px;
+  color: #ec5242;
+  padding: 0 1.5vw;
+}
+
+.headline .primary-text,
+.headline .secondary-text,
+.headline .join-us {
+  font-family: 'lato', sans-serif;
+}
+
+.headline .primary-text {
+  font-weight: 700;
+  font-size: 4vw;
+  color: #ec5242;
+  line-height: 23px;
+  padding: 0 1.5vw;
+}
+
+.headline .primary-text.about {
+  text-align: center;
+  padding: 0 4.5vw;
+}
+
+.headline .secondary-text {
+  font-weight: 300;
+  font-size: 3vw;
+  line-height: 17px;
+  padding: 10px 1vw;
+  background-color: aliceblue;
+}
+
+.headline .secondary-text.about {
+  padding: 0 0.5vw;
+}
+
+.headline .date {
+  font-weight: 200;
+  font-size: 7vw;
+  color: #272a31;
+}
+
+.headline .venue {
+  padding: 10px 5px;
+  font-weight: 400;
+  font-size: 5vw;
+  color: #272a31;
+}
+
+.web-address {
+  display: block;
+  padding: 10px 15px;
+  font-weight: 400;
+  font-size: 5vw;
+}
+
+.headline .contact-us {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 1.5vw;
+  line-height: 2.5vw;
+  text-align: center;
+  margin: 0 auto;
+  padding: 0 7.5vw;
+}
+
+.headline .email {
+  text-decoration: underline;
+  font-family: 'lato', sans-serif;
+  font-weight: 700;
+  color: #272a31;
+}
+
+/* main program card section */
+.main-program {
+  display: flex;
+  flex-direction: column;
+  background-image: url(../Images/program_bkgr_mob.png);
+  color: #fff;
+  padding: 7vw;
+}
+
+.main-heading {
+  position: relative;
+  margin-bottom: 10%;
+  font-family: 'lato', sans-serif;
+  font-weight: 600;
+  font-size: 5vw;
+  text-align: center;
+}
+
+.main-heading::after {
+  background: #ec5242;
+  content: '';
+  width: 10%;
+  height: 2px;
+  left: 43%;
+  bottom: -10px;
+  position: absolute;
+}
+
+.program-articles {
+  display: flex;
+  flex-direction: column;
+  gap: 3vw;
+}
+
+.main-program article {
+  display: flex;
+  padding: 3%;
+  gap: 5px;
+  align-items: center;
+  border: none;
+  background-image: url(../Images/article-program-mob.png);
+}
+
+.main-program .fa {
+  font-size: 7vw;
+}
+
```

Comment:
- [ ] **[Optional]** I think it would be better to  **align** elements on a the main program section on the mobile view to organize  and  to create connections between them and this will make the section **sharp** and **clear** as that of the [given design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

**Your design** | **Given design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/205445128-713c6f21-4e42-4af2-b634-98f0c652c558.png) | ![image](https://user-images.githubusercontent.com/102757126/205445210-ae325dd1-ddd2-4940-b2cf-f685c8f748e0.png)